TreeMaker/python/doZinvBkg.py

```diff
@@ -1,6 +1,91 @@
 import FWCore.ParameterSet.Config as cms
 
 def reclusterZinv(process, geninfo, residual, cleanedCandidates, suff, fastsim):
+    
+    ### AK8 detour
+
+    # https://twiki.cern.ch/CMS/JetToolbox
+    from JMEAnalysis.JetToolbox.jetToolbox_cff import jetToolbox
+    listBTagInfos = ['pfInclusiveSecondaryVertexFinderTagInfos'] 
+    listBtagDiscriminatorsAK8 = [
+        'pfCombinedInclusiveSecondaryVertexV2BJetTags',
+        'pfBoostedDoubleSecondaryVertexAK8BJetTags'
+    ]
+    jecLevels = ['L2Relative', 'L3Absolute']
+    if residual: jecLevels.append("L2L3Residual")
+    jetToolbox(process,
+        'ak8',
+        'jetSequence',
+        'out',
+        PUMethod = 'CHS',
+        miniAOD = True,
+        runOnMC = geninfo,
+        postFix='Clean',
+        newPFCollection = True,
+        nameNewPFCollection = 'cleanedCandidates',

```

Comment:
this should be `cleanedCandidates.value()` since `cleanedCandidates` is an input tag passed into this function (which could take any value)